client/landing/stepper/index.tsx

```diff
@@ -68,6 +68,8 @@ async function main() {
 	if ( redirectPathIfNecessary( pathname, search ) ) {
 		return null;
 	}
+	// Sympathy mode clears cache randomly, Stepper uses the cache to persist state (not really a cache).
```

Comment:
I'm commenting just so we don't forget to remove this 🤞🏻

Reply:
I added this on purpose. Why should we remove it?

Reply:
Lol sorry, I just woke up - I thought it was for testing purposes 😄